### ISSUE TITLE
Cursor becomes invisible after exiting notebook

### ISSUE THREAD

**Issue by remifan** [joouha/euporie#30]
Very impressive repo!

This minor issue is annoying that I have to enter then exit Tmux to recover the cursor.

This behaviour has been confirmed in Kitty, iTem2 and  Windows Terminal, so I believe it is not Terminal Emulator-related issue.

**Comment by joouha** [comment]
Thanks! And thanks for reporting this.

It seems that the terminal cursor is being left hidden when euporie exits, which shouldn't be happening. 

When you exit `tmux`, it will set the terminal cursor to visible, which is done using a [DECTCEM](https://vt100.net/docs/vt510-rm/DECTCEM.html) escape sequence. You could also set the cursor to visible with the following:

```bash
echo -e "\e[25h"
```

In `prompt_toolkit` (which euporie uses to interact with the terminal), the renderer should show the cursor when the application exits ([here](https://github.com/prompt-toolkit/python-prompt-toolkit/blob/04bb8bc05f577ee76c8b1896ceb78809acad4a93/src/prompt_toolkit/renderer.py#L259-L260)). But for some reason this does not seem to be working.

I shall investigate further!

**Comment by joouha** [comment]
I've found the problem - I had `erase_when_done` set to `True` for the notebook app.  It looks like `prompt_toolkit` will not reset the terminal if an application is full-screen and `erase_when_done` is set to `True`.  I reckon this is a bug in `prompt_toolkit` - I think `erase_when_done` should be ignored for full-screen applications.  I've [sumitted a PR](https://github.com/prompt-toolkit/python-prompt-toolkit/pull/1663) for this upstream.

I the mean-time, I've fixed this by setting `erase_when_done` to `False` in `euporie-notebook`.  If you'd like to test this fix in [`v2.0.3`](https://pypi.org/project/euporie/2.0.3/), I'd be grateful to hear if this has sorted the problem for you.
